### PR TITLE
User Show Page

### LINKED ISSUE #19: User Story 17, User Show Page can sort reviews
```
As a Visitor,
When I visit a user's show page
I should also see links to sort reviews in the following ways:
- sort reviews newest first (descending chronological order)
- sort reviews oldest first (ascending chronological order)
```

DO THESE STORIES FIRST:
- User Story 16, User Show Page

### Checklist:

- [x] base code tests are written
- [x] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [x] code is reviewed
- [x] branch is merged
- [x] all tests still pass

child of #12

child of #33

depends on #11

### PR DESCRIPTION
**Pull request by BrennanAyers**
This PR brings in the functionality of a User having their own show page where all their reviews are listed. These reviews include the Book cover, the Book title, the Date the Review was posted, the Review title, the Review rating, and finally, the Review text. Each of these are organized into their own `div`.
These reviews can also be sorted by their posted date. This is done via a Review class method, taking their `created_at` value from the database and sorting them before they hit the view.
Testing this method required time travel, provided by the ActiveHelper gem and the `travel_to` method.
Resolves #19

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'launchy'
   gem 'shoulda-matchers'
   gem 'simplecov'
+  gem 'activesupport'
```
Comment (alexander-mathieu) [inline Gemfile:43]: You'd think this would be installed by default by Rails!

File: spec/features/users/show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      click_link("Sort by: Newest")
+      expect(review_3.title).to appear_before(review_1.title)
+      expect(review_1.title).to appear_before(review_2.title)
+
+      click_link("Sort by: Oldest")
+      expect(review_2.title).to appear_before(review_1.title)
+      expect(review_1.title).to appear_before(review_3.title)
+    end
+  end
+end
```
Comment (alexander-mathieu) [inline spec/features/users/show_spec.rb:94]: Nice, test looks solid!

File: spec/models/review_spec.rb [hunk 3]
```diff
@@ -10,4 +10,48 @@
...
+
+      travel_to Time.zone.local(2019, 05, 11, 18, 00, 00)
+      @review_3 = @book_3.reviews.create!(title: "All the way home!", text: "This book went to the market!", rating: 3, user: @user)
+    end
+
+    it '.sort_by_date' do
+      expect(Review.sort_by_date("newest")).to eq([@review_3, @review_1, @review_2])
+      expect(Review.sort_by_date("oldest")).to eq([@review_2, @review_1, @review_3])
+    end
+  end
```
Comment (alexander-mathieu) [inline spec/models/review_spec.rb:56]: 👍
